README.md

```diff
@@ -189,6 +189,10 @@ To access a private container registry, follow the steps on [this page](http://k
 
 We recommend you use ImagePullSecrets, but if you would like to configure access on the minikube VM you can place the `.dockercfg` in the `/home/docker` directory or the `config.json` in the `/home/docker/.docker` directory.
 
+## Add-ons
+
+In order to have minikube properly start/restart custom addons, place the addon(s) you wish to be launched with minikube in the `.minikube/addons` directory.  Addons in this folder will be moved to the minikubeVM and launched each time minikube is started/restarted.
```

Comment:
Would it be helpful to include a link/reference to what an addon is, exactly?

e.g., the [source tree with the "canonical" list of addons](https://github.com/kubernetes/kubernetes/tree/master/cluster/addons), or [their definition in the k8s docs](http://kubernetes.io/docs/admin/cluster-components/#addons), maybe both?

Also, will any kubernetes addon work, or does minikube only support a subset of the kubernetes addons?